benchmarks/nemo/tracer_advection/original/README.md

```diff
@@ -12,14 +12,19 @@ for this library will have been obtained when you cloned the
 repository. The library is built automatically as part of the
 compilation process.
 
-The Makefile supports four targets:
-
-* tra_adv_serial  - the original, sequential form of the benchmark.
-* tra_adv_omp     - version transformed by PSyclone for OpenMP threading.
-* tra_adv_acc     - version transformed by PSyclone for OpenACC acceleration
-                    on GPU.
-* tra_adv_omp_gpu - version transformed by PSyclone for OpenMP offload to
-                    GPU.
+The Makefile supports the following targets:
+
+* tra_adv_serial         - the original, sequential form of the benchmark.
+* tra_adv_acc_kernels    - version transformed by PSyclone for OpenACC acceleration
+                           on GPU using the kernels directive.
+* tra_adv_acc_loops      - version transformed by PSyclone for OpenACC acceleration
+                           on GPU using the explicit loop directive.
+* tra_adv_omp_cpu_levels - version transformed by PSyclone for OpenMP threading
+                           on CPU parallelising loops over the k-domain.
+* tra_adv_omp_cpu        - version transformed by PSyclone for OpenMP threading
+                           on CPU parallelising all loops.
```

Comment:
Do you really mean *all* loops or "all outer loops"?

Reply:
I meant "all loop types" not just the ones over k. But I agree this is ambiguous, and I changed to your suggested "all outer loops".